### ISSUE TITLE
[NBI] Automatic sea ice segmentation in synthetic aperture radar images

### ISSUE THREAD

**Issue by louisavz** [eds-book-gallery/notebooks-ideas#1]
## What is the notebook about?

This notebook introduces the use of unsupervised learning on synthetic aperture radar (SAR) data for sea ice classification. In particular, we are interested in ice floes which are any contiguous piece of sea ice. The size of the floes within a region can provide important climatic information, and we aim to quantify this characteristic. The target geographical area is Weddell Sea of West Antarctica. The example here will focus only on Weddell Sea, although this should be scalable to other regions of interest if given the appropriate training dataset.

Packages used in this notebook will include core python functionalities, as well as
- [SciPy](https://scipy.org)
- [rasterio](https://rasterio.readthedocs.io/en/stable/)
- [scikit-learn](https://scikit-learn.org/stable/)
- [scikit-image](https://scikit-image.org)
- [OpenCV](https://opencv.org) (TBD)

This notebook will examine:
- What is SAR data
- Use of [SNAP toolbox](https://earth.esa.int/eogateway/tools/snap)
- Useful preprocessing steps for enhancing SAR image quality
- Example ways to create additional channels over the HH and HV cross-polarisation
- Walk through of the model
- Apply example test set to a pretrained model
- Examine the results when label data is either not available or poorly annotated (through visualisation)
- Some thoughts on what the results imply

This notebook will not examine:
- The preprocessing of SAR data using SNAP tool

## Data Science Component

- [x] Exploration
- [ ] Preprocessing
- [ ] Modelling
- [ ] Post-processing
- [ ] Other (e.g. Reproducibility): 

## Submission type

- [x] Standard
- [ ] Special Issue
- [ ] Other (e.g. CI2023 Reproducibility Challenge):

## Programming language

- [x] Python
- [ ] R
- [ ] Julia
- [ ] Other:

## Checklist:

- [x] Input data, pipeline and/or model are public with license/citation
- [ ] The proposed notebook reuses existing codebase
- [x] The proposed notebook uses open-source packages
- [x] The proposed notebook is associated to existing publication(s)

## Additional information 

Paper in preparation regarding existing publication(s).
